__tests__/wallet/api/swapService.test.ts

```diff
@@ -24,14 +24,14 @@ describe('hashing and encrypting', () => {
   it('should reject calls with missing parameters', () => {
     const originalString = 'PartialTx|123123||';
 
-    // @ts-ignore
+    // @ts-expect-error -- Testing invalid inputs
```

Comment:
Why not set this directive rule for the file?

Reply:
This would be a valid use of `ts-ignore`, as its scope is more ample. However, `ts-expect-error` is a directive valid for a single line and is aimed precisely at unit tests with invalid parameters.

You can learn more about it on [these release notes](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) for TypeScript 3.9.

In short, `ts-expect-error` acts as an additional `expect()` clause, but instead of testing the runtime it asserts that the next line would throw an error through a TypeScript compiler.